index.js

```diff
@@ -16,30 +23,46 @@ function moveToMapPosition(referenceMap, mapToMove) {
 // - could cause an infinite loop
 // - prematurely halts prolonged movements like
 //   double-click zooming, box-zooming, and flying
-function syncMaps(a, b) {
-  on();
+function syncMaps () {
+  var maps;
+  var argLen = arguments.length;
```

Comment:
Instead of relying on the `arguments` object, could we accept an array of maps instead? `arguments` introduces potential performance gotchas as well as indirection because it's not an array.